com.unity.multiplayer.mlapi/Runtime/Core/NetworkedObject.cs

```diff
@@ -395,8 +395,8 @@ public void Spawn(Stream spawnPayload = null, bool destroyWithScene = false)
 
             if (spawnPayload != null)
                 spawnPayload.Position = 0;
-
-            SpawnManager.SpawnNetworkedObjectLocally(this, SpawnManager.GetNetworkObjectId(), false, false, null, spawnPayload, spawnPayload != null, spawnPayload == null ? 0 : (int)spawnPayload.Length, false, destroyWithScene);
+            ulong netId =  SpawnManager.GetNetworkObjectId();
```

Comment:
what about `netObjId` to be more explicit?

Reply:
Actually reverting that back to how it is in the master branch.  SpawnManager.GetNetworkObjectId() replaces netId.